Library/Formula/dblatex.rb

```diff
@@ -0,0 +1,17 @@
+class Dblatex < Formula
+  env :userpaths
+  url "https://downloads.sourceforge.net/project/dblatex/dblatex/dblatex-0.3.5/dblatex-0.3.5.tar.bz2"
+  homepage "http://dblatex.sourceforge.net"
+  sha1 "3afd81cec40b2bacdf82ecf3901b3a89b73e2d6f"
+
+  depends_on "saxon"
+  depends_on :tex
+
+  def install
+    system "python", "setup.py", "install", "--prefix=#{prefix}", "--install-scripts=#{bin}"
```

Comment:
Typically python software is installed into a private location and only relevant executables are linked, see `ansible.rb` for an example.